src/shells.ts

```diff
@@ -1,30 +1,39 @@
 import * as vscode from 'vscode';
 
-export enum OS {
-  WINDOWS = 1,
-  LINUX = 2
-}
+// eslint-disable-next-line @typescript-eslint/naming-convention
+type systems = 'windows' | 'linux' | 'osx' | 'linuxbase';
```

Comment:
This could be expressed in a simpler way like this:

```ts
export enum OS {
  Windows,
  UnixLike
}
```

Then have a separate one for the defaultShells:

```ts
export enum OS_SHELL_KEY {
  Windows = 'windows',
  Linux = 'linux',
  MacOS = 'osx'
}
```